deps/make.blosc

```diff
@@ -89,3 +91,16 @@ $(LZ4_DIR)/lz4.o: $(LZ4_DIR)/lz4.c
 
 $(LZ4_DIR)/lz4hc.o: $(LZ4_DIR)/lz4hc.c
 	$(CC) $(CPPFLAGS) $(CFLAGS) -c -o $@ $(LZ4_DIR)/lz4hc.c
+
+$(SNAPPY_DIR)/snappy-c.o: $(SNAPPY_DIR)/snappy-c.cc
+	$(CC) $(CPPFLAGS) $(CXXFLAGS) -c -o $@ $(SNAPPY_DIR)/snappy-c.cc
```

Comment:
Is there a short description how to try the cross-compiling? I'd be happy to try to sort this out myself, just don't really know where to start. I have various linux virtual machines etc...